### PR TITLE
*: add batch operation for creating/dropping binding by specifying digests

### PR DESCRIPTION
**Pull request by time-and-fate**
### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/tidb/pull/55315

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by songrijie — APPROVED** [review]

File: sql-plan-management.md [hunk 1]
```diff
@@ -223,25 +223,25 @@ CREATE BINDING FOR SELECT * FROM t WHERE a > 1 USING SELECT * FROM t use index(i
 
 #### 根据历史执行计划创建绑定
 
-如需将 SQL 语句的执行计划固定为之前使用过的执行计划，可以使用 `plan_digest` 为该 SQL 语句绑定一个历史的执行计划。相比于使用 SQL 创建绑定的方式，此方式更加简便。
+如需将 SQL 语句的执行计划固定为之前使用过的执行计划，可以使用 Plan Digest 为该 SQL 语句绑定一个历史的执行计划。相比于使用 SQL 创建绑定的方式，此方式更加简便，并且可以支持一次为多个语句绑定执行计划。详细的说明和更多示例可以参考[CREATE [GLOBAL|SESSION] BINDING](/sql-statements/sql-statement-create-binding.md) 页面。
```
Comment (lilin90) [inline sql-plan-management.md:226]:
```suggestion
如需将 SQL 语句的执行计划固定为之前使用过的执行计划，可以使用 Plan Digest 为该 SQL 语句绑定一个历史的执行计划。相比于使用 SQL 创建绑定的方式，此方式更加简便，并且支持一次为多个语句绑定执行计划。详细说明和更多示例参见 [`CREATE [GLOBAL|SESSION] BINDING`](/sql-statements/sql-statement-create-binding.md)。
```

File: sql-plan-management.md [hunk 2]
```diff
@@ -223,25 +223,25 @@ CREATE BINDING FOR SELECT * FROM t WHERE a > 1 USING SELECT * FROM t use index(i
...
 
 以下为根据历史执行计划创建绑定的注意事项：
 
 - 该功能是根据历史的执行计划生成 hint 而实现的绑定，历史的执行计划来源是 [Statement Summary Tables](/statement-summary-tables.md)，因此在使用此功能之前需开启系统变量 [`tidb_enable_stmt_summary`](/system-variables.md#tidb_enable_stmt_summary-从-v304-版本开始引入)。
 - 对于包含子查询的查询、访问 TiFlash 的查询、3 张表或更多表进行 Join 的查询，自动生成的 hint 不够完备，可能导致无法完全固定住计划，对于这类情况在创建时会产生告警。
-- 原执行计划对应 SQL 语句中的 hint 也会被应用在创建的绑定中，如执行 `SELECT /*+ max_execution_time(1000) */ * FROM t` 后，使用其 `plan_digest` 创建的绑定中会带上 `max_execution_time(1000)`。
+- 原执行计划对应 SQL 语句中的 hint 也会被应用在创建的绑定中，如执行 `SELECT /*+ max_execution_time(1000) */ * FROM t` 后，使用其 Plan Digest 创建的绑定中会带上 `max_execution_time(1000)`。
 
 使用方式:
```
Comment (lilin90) [inline sql-plan-management.md:234]:
```suggestion
使用方式：
```

File: sql-statements/sql-statement-drop-binding.md [hunk 3]
```diff
@@ -15,18 +15,26 @@ aliases: ['/docs-cn/dev/sql-statements/sql-statement-drop-binding/']
 ```ebnf+diagram
 DropBindingStmt ::=
     'DROP' GlobalScope 'BINDING' 'FOR' ( BindableStmt ( 'USING' BindableStmt )?
-|   'SQL' 'DIGEST' SqlDigest)
+|   'SQL' 'DIGEST' StringLiteralOrUserVariableList)
```
Comment (lilin90) [inline sql-statements/sql-statement-drop-binding.md:18]: For format consistency.
```suggestion
|   'SQL' 'DIGEST' StringLiteralOrUserVariableList )
```

File: sql-statements/sql-statement-drop-binding.md [hunk 4]
```diff
@@ -15,18 +15,26 @@ aliases: ['/docs-cn/dev/sql-statements/sql-statement-drop-binding/']
...
     ( 'GLOBAL' | 'SESSION' )?
 
 BindableStmt ::=
     ( SelectStmt | UpdateStmt | InsertIntoStmt | ReplaceIntoStmt | DeleteStmt )
+
+StringLiteralOrUserVariableList:
+    (StringLitOrUserVariable | StringLiteralOrUserVariableList ',' StringLitOrUserVariable)
+
+StringLiteralOrUserVariable:
+    stringLiteral | UserVariable
```
Comment (lilin90) [inline sql-statements/sql-statement-drop-binding.md:30]: It seems that ebnf syntax error exists. You can preview it using [https://kennytm.github.io/website-docs/dist/](https://kennytm.github.io/website-docs/dist/).
```suggestion
StringLiteralOrUserVariableList ::=
    ( StringLitOrUserVariable | StringLiteralOrUserVariableList ',' StringLitOrUserVariable )

StringLiteralOrUserVariable ::=
    ( stringLiteral | UserVariable )
```

File: sql-statements/sql-statement-drop-binding.md [hunk 5]
```diff
@@ -15,18 +15,26 @@ aliases: ['/docs-cn/dev/sql-statements/sql-statement-drop-binding/']
...
 ```
 
 ## 示例
 
-你可以根据 SQL 语句或 `sql_digest` 删除绑定。
+你可以根据 SQL 语句或 SQL Digest 删除绑定。
+
+指定 SQL Digest 时，你既可以通过字符串字面量来指定，也可以通过字符串类型的用户变量来指定。你可以指定多个字符串，同时，每个字符串也可以包含多个 digest，它们之间均需要使用逗号隔开。
```
Comment (lilin90) [inline sql-statements/sql-statement-drop-binding.md:37]:
```suggestion
根据 SQL Digest 删除绑定时，你需要指定 SQL Digest，既可以通过字符串字面量来指定，也可以通过字符串类型的用户变量来指定。你可以指定多个字符串，同时每个字符串也可以包含多个 digest，字符串之间和 digest 之间均需使用逗号隔开。
```

**Review by lilin90 — APPROVED** [review]
/lgtm

File: sql-statements/sql-statement-create-binding.md [hunk 6]
```diff
@@ -17,21 +17,27 @@ aliases: ['/docs-cn/dev/sql-statements/sql-statement-create-binding/']
...
+
+StringLiteralOrUserVariable ::=
+    ( stringLiteral | UserVariable )
+```
 
 ## 示例
 
 你可以根据 SQL 或历史执行计划创建绑定。
 
+根据历史执行计划创建绑定时，你需要指定相应的 Plan Digest，既可以通过字符串字面量来指定，也可以通过字符串类型的用户变量来指定。你可以通过指定多个 Plan Digest 来同时为多个语句创建绑定，此时你可以指定多个字符串，同时每个字符串也可以包含多个 digest，注意字符串之间和 digest 之间均需使用逗号隔开。
```
Comment (hfxsd) [inline sql-statements/sql-statement-create-binding.md:39]:
```suggestion
你可以根据 SQL 或历史执行计划创建绑定。根据历史执行计划创建绑定时，你需要指定相应的 Plan Digest，此时：

- 既可以通过字符串字面量来指定，也可以通过字符串类型的用户变量来指定。
- 可以通过指定多个 Plan Digest 来同时为多个语句创建绑定，此时你可以指定多个字符串，同时每个字符串也可以包含多个 digest，注意字符串之间和 digest 之间均需使用逗号隔开。
```

**Review by hfxsd — APPROVED** [review]

**Pull request merged**